src/jvmMain/kotlin/io/rippledown/kb/KB.kt

```diff
@@ -11,14 +11,14 @@ class KB(val name: String) {
 
     fun startSession(case: RDRCase, action: RuleTreeChange): RuleBuildingSession {
         //interpret the case and all cornerstones at the start of each session
-        cornerstones.forEach { interpret(it) }
-        interpret(case)
-        return RuleBuildingSession(case, action, cornerstones)
+//        cornerstones.forEach { interpret(it) }
```

Comment:
maybe delete these lines and the comment above as it looks like you re-interpret the CCs elsewhere